taxi/tests/test_form.py

```diff
@@ -0,0 +1,41 @@
+from django.test import TestCase
+
+from taxi.forms import (
+    DriverCreationForm,
+    DriversSearchForm,
+    CarsSearchForm,
+    ManufacturerSearchForm
+)
+
+
+class FormsTest(TestCase):
+    def test_driver_creation_form_with_license_number_name_is_valid(self):
+        form_data = {
+            "username": "new_user",
+            "password1": "user12test",
+            "password2": "user12test",
+            "first_name": "test first",
+            "last_name": "test last",
+            "license_number": "TES12345",
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+        self.assertEqual(form.cleaned_data, form_data)
+
+    def test_drivers_search_form(self):
```

Comment:
Here you are just testing if form data is valid. For correct testing you should add a bigger dataset for each model and test that search result Contains records that should be there and NotContains records that need to be filtered out